api/src/main/java/gov/va/vro/api/resources/ClaimMetricsResource.java

```diff
@@ -149,4 +149,37 @@ ResponseEntity<List<ExamOrderInfoResponse>> allExamOrderInfo(
       @RequestParam(name = "size", required = false, defaultValue = "10")
           @Min(value = 1, message = "invalid size")
           Integer size);
+
+  @Operation(
+      summary = "Retrieves all exam order records",
```

Comment:
Might want to mention slack in the summary just so people dont accidentally spam slack.